src/promptflow-evals/tests/evals/e2etests/test_evaluate.py

```diff
@@ -132,14 +132,16 @@ def test_evaluate_with_content_safety_evaluator(self, project_scope, data_file):
         assert 0 <= metrics.get("content_safety.self_harm_defect_rate") <= 1
         assert 0 <= metrics.get("content_safety.hate_unfairness_defect_rate") <= 1
 
-    def test_evaluate_python_function(self, data_file):
+    @pytest.mark.parametrize('use_thread_pool', [True, False])
+    def test_evaluate_python_function(self, data_file, use_thread_pool):
         # data
         input_data = pd.read_json(data_file, lines=True)
 
         # run the evaluation
         result = evaluate(
             data=data_file,
             evaluators={"answer": answer_evaluator},
+            use_thread_pool=use_thread_pool
```

Comment:
I renamed to _use_thread_pool as an internal use parameter

Reply:
Fixed